Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Hmeg on 2023/06/06.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let operandQueue = CalculatorItemQueue<Double>()
+        let operatorQueue = CalculatorItemQueue<Operator>()
+        
+        let components = componentsByOperators(from: input)
+        
+        components.compactMap { Double($0) }.forEach { operandQueue.enqueue(item: $0)}
```

Comment:
```suggestion
        let components = componentsByOperators(from: input)
        
        components
            .compactMap { Double($0) }
            .forEach { operandQueue.enqueue(item: $0) }
```
1. 다른 곳과 같이 chaining을 통해 고차함수를 사용해보는 것은 어떨까요?
2. 고차함수를 쓰실 때, 컨벤션이 다른 경우가 많은 것 같아요. 특히 띄어쓰기를 일정하게 유지하는게 좋을 것 같아요.